docs/configure-zapi.md

```diff
@@ -189,3 +189,76 @@ exporters, the following parameters can be defined:
 * `instances_keys` (list): display names of labels to export with each data-point
 * `instance_labels` (list): display names of labels to export as a separate data-point
 * `include_all_labels` (bool): export all labels with each data-point (overrides previous two parameters)
+
+### Filter
+
+This guide provides detailed instructions on how to use the `filter` feature in ZapiPerf.
+
+#### Objects (Excluding Workload)
+
+In ZapiPerf templates, you can set up filters under `counters`. Wildcards like * are useful if you don't want to specify all instances.
```

Comment:
Please note, ONTAP Zapi filtering does not support regular expressions, only wildcard matching with * and ?.

Reply:
Check if `?` works and if not remove that clause